tests/pytorch_pfn_extras_tests/onnx/test_annotate.py

```diff
@@ -0,0 +1,317 @@
+from collections import OrderedDict
+from contextlib import nullcontext
```

Comment:
nullcontext is from 3.7 https://docs.python.org/3/library/contextlib.html#contextlib.nullcontext